### PR TITLE
feat: add option to show disk usage of multiple disks

### PR DESCRIPTION
**Pull request by Rolv-Apneseth**
In relation to #286 

After rebasing the `8.0.0` branch over on `libmacchina` (see Macchina-CLI/libmacchina#179), and cherry-picking an old commit I had where I achieved the disks functionality, it all seems to still work. Let me know what you think.

Note that the `Cargo.toml` and `Cargo.lock` changes aren't included since `8.0.0` isn't live, and I just pointed `libmacchina` to a local path for development.

### CODE REVIEW

**Comment by grtcdr** [comment]
Thanks for working on this, Rolv.

I merged the 8.0.0 rebase branch over in libmacchina.

> Note that the Cargo.toml and Cargo.lock changes aren't included since 8.0.0 isn't live, and I just pointed libmacchina to a local path for development.

Feel free to change the two files, `master` branch is a moving target and since NetBSD is in a rough patch with the current state of `disk_space`, it's best if we started testing these changes directly.

**Comment by grtcdr** [comment]
> Just when you get the chance to update libmacchina and review this, would you also be interested in some other stuff I worked on in my own fork, such as options for shorter CPU and GPU names? @grtcdr

Sure, I'd love to take a look :)

**Comment by Rolv-Apneseth** [comment]
> `master` branch is a moving target so feel free to change the two files.

Alright I have updated the files - though until `libmacchna` v8 is released the tests won't pass. Would you like me to bump the version of `macchina` itself? Also maybe a changelog entry, or do you prefer to do those yourself?



> Sure, I'd love to take a look :)

Great, I'll have a look at making some PRs after this one is finished then

**Comment by grtcdr** [comment]
> Would you like me to bump the version of macchina itself? Also maybe a changelog entry, or do you prefer to do those yourself?

All that would be great.

Thanks for all the help :)

**Review by grtcdr — APPROVED** [review]

**Comment by Rolv-Apneseth** [comment]
Made some minor changes there but how do you run `scdoc` again? I did `scdoc < macchina.1.scd > macchina.1` and it made a lot more changes than I intended.

**Comment by grtcdr** [comment]
That's alright. I ran the same command on my machine and it generated identical changes, for example: `.P` -> `.PP`. `scdoc` must have changed its output in recent versions.

**Comment by Rolv-Apneseth** [comment]
Alright cool

File: Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 build = "build.rs"
 
 [dependencies]
-libmacchina = { version = "7.3.1", features = ["version"] }
+libmacchina = { version = "8.0.0", features = ["version"] }
```
Comment (grtcdr) [inline Cargo.toml:14]: This should be changed to:

```
libmacchina = { git = "https://github.com/Macchina-CLI/libmacchina", branch = "8.0.0", features = ["version"] }
```

**Review by grtcdr — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (Rolv-Apneseth) [reply Cargo.toml:14]: Oh sure yeah, wasn't sure you wanted that. I'll update it

**Comment by grtcdr** [comment]
The readout's accuracy has definitely improved but there is still a discrepancy between the results of `macchina` and those of `lsblk`, here's an example:

```
~/projects/macchina $ grep disk ~/.config/macchina/macchina.toml
disks=["/"]

~/projects/macchina $ cargo run
                                 
  Disk Space   96.4 GB / 337.1 GB       

~/projects/macchina $ lsblk         
NAME          MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINTS
nvme0n1       259:0    0 953.9G  0 disk  
├─nvme0n1p1   259:1    0   512M  0 part  /boot
└─nvme0n1p2   259:2    0 953.4G  0 part  
  ├─lvm-root  254:0    0   320G  0 lvm   /gnu/store
  │                                      /
  ├─lvm-swap  254:1    0     8G  0 lvm   [SWAP]
  ├─lvm-state 254:2    0    12G  0 lvm   [SWAP]
  └─lvm-home  254:3    0   480G  0 lvm   
    └─home    254:4    0   480G  0 crypt /home
```

**Comment by Rolv-Apneseth** [comment]
I didn't touch how the disk space readout is calculated on this one - but did you use `df -H`? It's pretty spot on for me

**Comment by grtcdr** [comment]
Totally forgot we were doing powers of 1000!

Yeah, the results are accurate now :)

**Comment by Rolv-Apneseth** [comment]
All good. Powers of 10 are the standard as far as I know, just some old stuff still uses power of 2 when the units for power of 2 should be `KiB`, `MiB` etc.

**Comment by grtcdr** [comment]
Alright, well this looks good to me :)

Thanks a lot for the terrific work!

**Comment by Rolv-Apneseth** [comment]
No problem!

**Pull request merged**